Modules/Dining/MITDiningHouseMealSelectorPad.m

```diff
@@ -353,4 +350,55 @@ - (NSString *)stringForMealDate:(NSDate *)date
     }
 }
 
+#pragma mark - Touches
+
+- (void)touchesBegan:(NSSet *)touches withEvent:(UIEvent *)event
+{
+    [self handleTouch:touches.anyObject isDragging:NO];
+}
+
+- (void)touchesMoved:(NSSet *)touches withEvent:(UIEvent *)event
+{
+    [self handleTouch:touches.anyObject isDragging:YES];
+}
+- (void)touchesEnded:(NSSet *)touches withEvent:(UIEvent *)event
+{
+    [self updateForCurrentlySelectedLetterView];
+    [UIView animateWithDuration:0.25 animations:^{
+        self.selectedMealBackground.transform = CGAffineTransformMakeScale(1.0, 1.0);
+        self.currentlyHighlightedLetterView.transform = CGAffineTransformMakeScale(1.0, 1.0);
+    }];
+}
+
```

Comment:
You should also implement the `touchesCanceled:` method to clean up the UI in case something happens that cancels the touch. Probably the same code as in `touchesEnded:`